tools/coverage/calc/base.go

```diff
@@ -147,7 +147,7 @@ func (c *Coverage) IsCoverageLow(covThresholdInt int) bool {
 	if err == nil {
 		return ratio < covThreshold
 	}
-	return true
+	return false
```

Comment:
suggest add a comment explaining the logic

Reply:
1. Go file with no statement should not be marked as low coverage file.

Reply:
sorry for not making it clear, I was suggesting adding a comment above this function explaining when this returns true and when this returns false